### ISSUE TITLE
Comparison of systems

### ISSUE THREAD

**Issue by arpita2512** [maximedb/full#1]
Hi

Just want to confirm that when comparing 2 systems/conversations, the one producing a lower score is better, right? Since you're evaluating the log likelihood of negative follow-ups.

Thanks

**Comment by maximedb** [comment]
Hi,

The correlation with human evals is positive, so a higher score is better. This is because of the `torch.exp(loss)` here: https://github.com/maximedb/full/blob/c66eba64cca6c434ebb6ddefca5cd518bb2090d4/full/eval.py#L51C16-L52C1
